### ISSUE TITLE
Installing through pip without -e leaves ots unusable

### ISSUE THREAD

**Issue by aisopuro** [hegenator/ots#4]
Steps to repeat:
1. Make sure you don't have an installation of `ots` working (I removed both `~/.local/bin/ots` and `/.local/lib/python3.6/site-packages/ots`)
2. pip install from the github repo: `pip3 install --user --upgrade git+https://github.com/hegenator/ots.git@dev`
3. Run `ots` (without arguments)

This results in a traceback:
```
~ $ ots 
Traceback (most recent call last):
  File "/home/atte/.local/bin/ots", line 5, in <module>
    from ots.cli import cli
  File "/home/atte/.local/lib/python3.6/site-packages/ots/cli.py", line 11, in <module>
    from .migration.migrate import check_and_migrate
ModuleNotFoundError: No module named 'ots.migration'
```

Removing again and installing with `pip3 install --user -e github/ots` makes it so it works.

**Comment by hegenator** [comment]
This is probably because I've forgotten to include the migration module in the setup.py, that would be my guess. I'll look into it

**Comment by hegenator** [comment]
This has been fixed in commit 725bafaa3deb56f18759803295085acfd0829a9f

Closing the issue.
